developers/weaviate/modules/reader-generator-modules/generative-anyscale.md

```diff
@@ -250,40 +250,57 @@ import AWSSingleResult from '/_includes/code/generative.aws.singleresult.mdx';
 #### Example - grouped result
 
 Here is an example of a query where:
-* we run a vector search (with `nearText`) to find publications about finance,
-* then we ask the generator module to explain why these articles are about finance.
+* we run a vector search (with `nearText`) to find podcast clips semantically similar to `"What is ref2vec?"`
```

Comment:
Consider parallel list structure:

* We run ... ref2vec?"`.
* We ask ... results.

Reply:
Hey @ this is more or less verbatim the same as the other `generative` module examples. So I'll see if we can refactor all of them together.